test-cursors/Main.hs

```diff
@@ -0,0 +1,18 @@
+module Main where
+
+import           Test.Tasty
+
+import qualified Test.Database.LMDB.Simple.Cursor as Test.Cursor
+
+main :: IO ()
+main = defaultMain $ testGroup "test-cursors" [
+    testGroup "Test" [
```

Comment:
Regarding my "bench/Bench" question, here we see that we only have one "test" level. Whatever we pick maybe we should be consistent.

Reply:
See https://github.com/input-output-hk/lmdb-simple/pull/1#discussion_r1029320398